packages/cell-toolbar/src/celltoolbartracker.ts

```diff
@@ -322,13 +365,17 @@ export class CellToolbarTracker implements IDisposable {
       return false; // Nothing in the editor
     }
 
-    const codeMirrorLines =
-      editorWidget.node.getElementsByClassName('CodeMirror-line');
+    const codeMirrorLines = editorWidget.node.getElementsByClassName('cm-line');
     if (codeMirrorLines.length < 1) {
       return false; // No lines present
     }
-    const lineRight = codeMirrorLines[0].children[0] // First span under first pre
-      .getBoundingClientRect().right;
+
+    let lineRight = codeMirrorLines[0].getBoundingClientRect().left;
+    const range = document.createRange();
+    range.setStart(codeMirrorLines[0].childNodes[0], 0);
+    range.selectNodeContents(codeMirrorLines[0]);
+    const sizes = range.getClientRects();
+    lineRight += [...sizes].reduce((prev, rect) => prev + rect.width, 0);
```

Comment:
Let's try this:

```suggestion
    range.selectNodeContents(codeMirrorLines[0]);
    lineRight += range.getBoundingClientRect().width;
```

After debugging the failures it seems that Chrome double-counts some nodes in the range when using `getClientRects` approach. The proposed approach works when testing in-browser.